api-reference/10 UI Widgets/dxScheduler/3 Methods/scrollTo(date_group_allDay).md

```diff
@@ -3,16 +3,26 @@ id: dxScheduler.scrollTo(date, group, allDay)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Scrolls the current view to a specified date, time, and optionally, appointment group. Requires the [height](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#height) property to be set.
 
 ##### param(date): Date
-<!-- Description goes here -->
+A date and time.
 
 ##### param(group): Object | undefined
-<!-- Description goes here -->
+Specifies the appointment group if any.
 
 ##### param(allDay): Boolean | undefined
-<!-- Description goes here -->
+If **true**, scrolls to the top of the view. Appliable for the *"day"* and *"week"* [views](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/) if [showAllDayPanel](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#showAllDayPanel) is enabled.
```

Comment:
> Appliable for the *"day"* and *"week"* [views](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/) if [showAllDayPanel](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#showAllDayPanel) is enabled.

Applies to workweek view as well. Anyway, it's a bit hard to keep all these prerequisites in mind. Let's just say as following:

```suggestion
If **true**, scrolls the view to the all-day panel. Applies only if the all-day panel is [visible](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#showAllDayPanel).
```